File: 01_ProgrammingFundamentals/Homeworks/07_Objects and Classes-Lab/06_Rectangle Position/Rectangle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _06_Rectangle_Position
{
    public class Rectangle
    {
        public double left { get; set; }
        public double top { get; set; }
        public double width { get; set; }
        public double height { get; set; }

        public double bottom => top - height;
        public double right => bottom + width;
    }
}

File: 07_OOP_Advanced/13_OpenClosedAndLiskovSubstitution_lab/src/p03_GraphicEditor/Rectangle.java
package p03_GraphicEditor;

public class Rectangle extends Shape {
	
	private Double width;
	private Double height;
	
	public Rectangle(Double width, Double height) {
		this.width = width;
		this.height = height;
	}

	@Override
	public void drow() {
		System.out.println("I'm a Rectangle!");
	}
	
}
